.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+
+services:
+  - docker
+
+langauge: go
+
+go:
+  - 1.13.x
+  - 1.14.x
```

Comment:
Two versions of Go?

Reply:
We've had some issues with 1.14 in goland (debugger not working, strange test runner behaviour) so have been sometimes using 1.13 for local developlment, but the docker image is building with the latest 1.14.

Reply:
shouldn't we tell travis to just use one version though?